samples/common.mk

```diff
@@ -0,0 +1,18 @@
+mkfile_dir=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
Do you need to redefine common.mk? Can you just include ../tests/common.mk ?

Reply:
Done.